src/index.html

```diff
@@ -7,8 +7,25 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#">
+        <div class="img"></div>
+      </a>
+      <nav>
+        <ul>
+          <li><a class="is-active link" href="#">Apple</a></li>
+          <li><a class="link" href="#">Samsung</a></li>
+          <li><a class="link" href="#">Smartphones</a></li>
+          <li><a data-qa="hover" class="link">Laptops & Computers</a></li>
+          <li><a class="link" href="#">Gadgets</a></li>
+          <li><a class="link" href="#">Tablets</a></li>
+          <li><a class="link" href="#">Photo</a></li>
+          <li><a class="link" href="#">Video</a></li>
```

Comment:
Fix formatting according to https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md code style rules